app/views/projects/show.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <h2>Project Details</h2>
 
-<% for attribute in @project.attributes.keys %>
+<% @project.attributes.keys.each do |attribute| %>
   <% if attribute != 'id' and attribute != 'created_at' and attribute != 'updated_at' %>
```

Comment:
Conceptually, what we're trying to iterate over is the attributes that should get printed out. In this case, this means all the attributes that aren't `id`, `created_at`, or `updated_at`. Typically we'd prefer to write something like

```ruby
@project.attributes_names.reject { |attribute| ['id', 'created_at', 'updated_at'].include? attribute }
```

But it's prolly worth asking: will we need this list of attributes elsewhere? If so, then it's worth thinking about ways to DRY this out. For example, we can write a method in the model class, in which case this line might look like:

```ruby
@project.readable_attributes.each do |attribute|
```

This might seem a bit contrived but it serves to illustrate the more general question of where to put things. No need to change anything here for now but keep an eye out for this.